requirements.txt

```diff
@@ -1,33 +1,4 @@
-beautifulsoup4==4.12.2
-bleach==6.0.0
-build==0.10.0
-cachetools==5.3.2
-certifi==2023.7.22
-charset-normalizer==3.2.0
-docutils==0.20.1
-idna==3.4
-importlib-metadata==6.8.0
-jaraco.classes==3.3.0
-keyring==24.2.0
-markdown-it-py==3.0.0
-mdurl==0.1.2
-more-itertools==10.1.0
-packaging==23.1
-pkginfo==1.9.6
-pycryptodomex==3.19.0
-Pygments==2.16.1
-pyproject_hooks==1.0.0
-python-decouple==3.8
-readme-renderer==41.0
-requests==2.31.0
-requests-toolbelt==1.0.0
-rfc3986==2.0.0
-rich==13.5.2
-six==1.16.0
-soupsieve==2.5
-steam==1.4.4
-twine==4.0.2
-urllib3==2.0.4
-vdf==3.4
-webencodings==0.5.1
-zipp==3.16.2
+beautifulsoup4
+charset-normalizer
+python-decouple
+requests
```

Comment:
Niiiceee... Probably did a pip freeze on a different python env. Thanks for updating